proto/regen/ecocredit/v1/query.proto

```diff
@@ -559,11 +589,33 @@ message QueryCreditTypesResponse {
 // QueryParamsRequest is the Query/Params request type.
 message QueryParamsRequest {}
 
+// AllowedDenom represents the information for an allowed ask denom.
+message AllowedDenomInfo {
+  // denom is the bank denom to allow (ex. ibc/GLKHDSG423SGS)
+  string bank_denom = 1;
+
+  // display_denom is the denom to display to the user and is informational.
+  // Because the denom is likely an IBC denom, this should be chosen by
+  // governance to represent the consensus trusted name of the denom.
+  string display_denom = 2;
+
+  // exponent is the exponent that relates the denom to the display_denom and is
+  // informational
+  uint32 exponent = 3;
+}
+
 // QueryParamsResponse is the Query/Params response type.
 message QueryParamsResponse {
 
   // params defines the parameters of the ecocredit module.
   Params params = 1;
+
+  // allowed_denoms is a list of bank denoms allowed to be used in the ask price of
+  // sell orders.
+  //
+  // Since Revision 1
+  repeated AllowedDenomInfo allowed_denoms = 2;
```

Comment:
```suggestion
  // allowed_denoms is a list of bank denoms allowed to be used in the ask price of
  // sell orders.
  //
  // Since Revision 1
  repeated AllowedDenom allowed_denoms = 2;
```